business_requirement_crm/models/business.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+        ondelete='set null',
+    )
+    resource_cost_total = fields.Float(
+        compute='_get_resource_cost_total',
+        string='Total Revenue from BR'
+    )
+
+    @api.one
+    def _get_resource_cost_total(self):
+        linked_brs = self.project_id and self.project_id.br_ids
```

Comment:
@elicoidal This is also deprecated the `br_ids` feature is no longer introduced by base module `business_requirement`. I guess I should make this module depends on the `business_requirement_project`. What do you think?